broker/tasks/iam.py

```diff
@@ -135,18 +135,31 @@ def delete_previous_server_certificate(operation_id: str, **kwargs):
     else:
         iam = iam_govcloud
 
-    for certificate in Certificate.query.filter(
+    certificates_to_delete = Certificate.query.filter(
         and_(
             Certificate.service_instance_id == service_instance.id,
             Certificate.id != service_instance.current_certificate_id,
         )
-    ).all():
+    ).all()
+    for certificate in certificates_to_delete:
         try:
+            print(certificate.iam_server_certificate_name)
```

Comment:
this should probably go through logging, right? or was it supposed to be deleted?

Reply:
yeah, this was for debugging. i'll remove